solution/ui/regulations/eregs-component-lib/src/components/index.js

```diff
@@ -32,6 +32,14 @@ export { default as CopyCitation } from "./tooltips/CopyCitation.vue";
 export { default as GovInfoLinks } from "./tooltips/GovInfoLinks.vue";
 export { default as TooltipContainer } from "./tooltips/TooltipContainer.vue";
 
+// Shared Components
+export { default as LeftNavCollapse } from "./shared-components/LeftNavCollapse.vue";
+export { default as Toc } from "./shared-components/TOC/Toc.vue";
+export { default as TocContainer } from "./shared-components/TOC/TocContainer.vue";
+export { default as TocPart } from "./shared-components/TOC/TocPart.vue";
+export { default as TocTitle } from "./shared-components/TOC/TocTitle.vue";
+export { default as TocSubchapter } from "./shared-components/TOC/TocSubchapter.vue";
+
```

Comment:
A longer-term vision for front end organization is to have all components in a single location, and allow them to be used either in the `eregs-component-lib` project or the `eregs-vite` project.  This is the first step towards that goal.  Ideally these components would exist outside of both projects, but there are issues with Vuetify that will need to be fixed first. So for now they're here.